scripts/sct_detect_pmj.py

```diff
@@ -217,11 +211,10 @@ def extract_sagital_slice(self):
     def orient2pir(self):
         """Orient input data to PIR orientation."""
```

Comment:
not sure why this "PIR" suddenly comes in, but it has nothing to do with this PR anyways...